.travis.yml

```diff
@@ -1,18 +0,0 @@
-dist: trusty
```

Comment:
Maybe we should first validate the new GitHub Action flow on a couple of PRs before deleting this?